docker/filesystem/compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3.8'
+services:
+  zookeeper:
+    image: "confluentinc/cp-zookeeper:7.3.3"
+    ports:
+      - "2181:2181"
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
```

Comment:
We can skip running a separate ZooKeeper container if we use `run.sh` as in the demo repo. Should we maybe?

Reply:
I think it's simpler to use the containers as is to avoid adding another script to maintain; but if you feel strongly we can surely add it.

Reply:
Ok, let's keep as is